workloads/opea/chatqna/README.md

```diff
@@ -0,0 +1,91 @@
+# Deploy OPEA ChatQnA workload on OCP
+
+## Overview
+The workload is based on the [OPEA ChatQnA Application](https://github.com/opea-project/GenAIExamples/tree/v0.8/ChatQnA) running on Intel® Gaudi Accelerator with OpenShift and OpenShift AI. Refer to the [OPEA Generative AI Examples](https://github.com/opea-project/GenAIExamples/tree/v0.8) for more details about the OPEA workloads. 
+ 
+## Prerequisites
+* Provisioned RHOCP cluster. Follow steps [here](/README.md#provisioning-rhocp-cluster)
+* The Persistent storage using NFS is ready. Refer to [documentation](https://docs.openshift.com/container-platform/4.16/storage/persistent_storage/persistent-storage-nfs.html) for the details to set it up.
+
+**Note**: Refer to [documentation](https://docs.openshift.com/container-platform/4.16/storage/index.html) for setting up other types of persistent storages.
+* Provisioned Intel Gaudi accelerator on RHOCP cluster. Follow steps [../../../gaudi/README.md]
+* Installed and Configured RHOAI. Follow steps [here](../inference/README.md/#install-rhoai) 
+* Minio based S3 service ready for RHOAI. Follow steps [here](https://ai-on-openshift.io/tools-and-applications/minio/minio/#create-a-matching-data-connection-for-minio)
+* Create the Persistent Volume for the Application
```

Comment:
## Create Persistent Volumes
The NFS is used to create the Persistent Volumes for ChatQnA MegaService to claim and use.
Make sure to update NFS server IP and path in  ```persistent_volumes.yaml``` before applying command below. 
For example:
```
  nfs:
    server: 192.168.0.21 # nfs server
    path: /my_nfs # nfs path
```

``` 
$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/workloads/opea/chatqna/persistent_volumes.yaml
```

Reply:
> ## Create Persistent Volumes
> The NFS is used to create the Persistent Volumes for ChatQnA MegaService to claim and use. Make sure to update NFS server IP and path in `persistent_volumes.yaml` before applying command below. For example:
> 
> ```
>   nfs:
>     server: 192.168.0.21 # nfs server
>     path: /my_nfs # nfs path
> ```
> 
Updated
> @vbedida79 please add the command to verify the Persistent Volumes
the command below  shows both persistent volumes and persistent volumes created and bound. this verifies both.